presto-hive/src/test/java/io/prestosql/plugin/hive/util/TestOrcWriterOptions.java

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
huawei copyright header needs to be added

Reply:
fixed